components/common/ContentBox.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTheme, Box } from '@chakra-ui/react'
+
+export const RESPONSIVE_PADDING = [4, null, 8]
+
+interface Props {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  py?: any
```

Comment:
Anyone know if there's a better way to do this? TypeScript is not my friend.

I want to allow any prop that `Box` would take, and pass those through to it. But, also, allow a boolean true value for `py` (see use in `StoryFeed` below), and use the `RESPONSIVE_PADDING` constant in that case. Chakra's type for `py` is super complicated and I don't want to repeat all that. Can I refer to it somehow in a union that adds `true` somehow?

Reply:
I think you can `extend` a type. Or do like `type CustomBoxProps = NewProp & BoxProps` type thing (that's pseudocode). Can you import the type for `Box` from somewhere? I poked at it, but couldn't find it.

Reply:
I couldn't find a type for `py` to import, just a long inline definition with other types that I would have had to imported from who-knows-where. It seemed way overkill for this, but I'm wondering if there's something simpler that I missed.

Reply:
oh, I see. I misunderstood what `py` was. I'll pull this down and play with it. Although maybe we roll with this for now? Feels like a pretty low risk thing to be telling the linter to ignore.

Reply:
I agree. It seems very low risk considering how many different types of data are valid in that context. You're probably more likely to pass something valid that's not what you intended than something altogether invalid.